.travis.yml

```diff
@@ -16,9 +16,9 @@ stages:
   - name: build_release
     if: branch =~ /^rel\// AND type != pull_request
   - name: deploy
-    if: branch =~ /^rel\/nightly/ AND type != pull_request
+    if: (branch =~ /^rel\/nightly/ OR branch =~ /^rel\/dev/) AND type != pull_request
   - name: release
-    if: branch =~ /^rel\/nightly/ AND type != pull_request
+    if: (branch =~ /^rel\/nightly/ OR branch =~ /^rel\/dev/) AND type != pull_request
```

Comment:
I left this comment on https://github.com/algorand/go-algorand-internal/pull/3
> To support more release branches than just rel/nightly, we should specifically exclude rel/stable rather than only include nightly:
> 
> This seems to work: branch != rel/stable AND branch =~ /^rel\//
> ```
> $ travis-conditions eval "branch != rel/stable AND branch =~ /^rel\//" --data '{"branch": "rel/stable"}'
> false
> $ travis-conditions eval "branch != rel/stable AND branch =~ /^rel\//" --data '{"branch": "rel/nightly"}'
> true
> $ travis-conditions eval "branch != rel/stable AND branch =~ /^rel\//" --data '{"branch": "master"}'
> false
> ```

Reply:
Additionally, we could use the travis deployments for this rather than tasks. I'm  using the github releases provider for SHD but we could use the script or s3 provider as well:
https://docs.travis-ci.com/user/deployment/script/
https://docs.travis-ci.com/user/deployment/s3/